src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -22,4 +22,9 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="5.0.0-preview.7.20308.13" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Also you can take this file from AI: https://github.com/microsoft/ApplicationInsights-dotnet/blob/4a64eaf834ee4741fb2014b8dfef8426eb7e6ca0/.scripts/tool_FinalizePublicApi.ps1

Reply:
just took and updated so it will work for our current structure.